.mvn/maven.config

```diff
@@ -1,2 +1,3 @@
 -Pconsume-incrementals
 -Pmight-produce-incrementals
+-Duse-jenkins-bom
```

Comment:
What is this for?

Reply:
It's a magic "fix dependency problems" switch 🤷 Maybe it's no longer needed, will check.

Reply:
I checked the current version of `plugin-pom` and did not see it, so I was not sure what it was for specifically. Maybe it was required in the beta 4.x releases to enable the Jenkins BOM? I think I fixed some version conflicts with core dependencies by updating to 4.x versions of the parent POM on other plugins without needing to add this, although maybe that was something else.